common/adder/adder.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8"?>
+<component version="1.4" xmlns="http://schemas.circuit-diagram.org/circuitDiagramDocument/2012/component/xml">
+  <declaration>
+    <meta name="name" value="Adder" />
+    <meta name="minsize" value="50" />
+    <meta name="author" value="Corsaka" />
+    <meta name="guid" value="8f764035-18c6-4421-ac7f-5cbd327d4e78" />
+
```

Comment:
It looks like this when placed vertically:

![image](https://user-images.githubusercontent.com/5303088/75490952-01a49900-59ad-11ea-8b9c-61b2a45045b8.png)

Could you add the following to only allow it to be placed horizontally?

```xml
<flags>
  <option>horizontalonly</option>
</flags>
```